documentation/modules/ROOT/pages/transformations/timezone-converter.adoc

```diff
@@ -0,0 +1,210 @@
+:page-aliases: configuration/timezone-converter.adoc
+[id="timezone-converter"]
+= Timezone Converter
+
+:toc:
+:toc-placement: macro
+:linkattrs:
+:icons: font
+:source-highlighter: highlight.js
+
+toc::[]
+
+[[timezone-converter-introduction]]
+
+When {prodname} emits event records, the timezone values for timestamp fields in the record can vary depending on the type and configuration of the data source.
+To maintain data consistency and precision within data processing pipelines and applications, you can use the `Timezone Converter` SMT to convert the time zone values of timestamp fields so that they represent data consistently.
+It is specifically helpful when you need to ensure that timestamps in your event data are consistent in timezone representation.
+
+The SMT converts the value of the specified field to the target timezone by using the `converted.timezone` configuration option.
+You can specify the target timezone as a geographic timezone, for example,  `America/New_York`, or as a UTC offset, such as `+02:00`.
+It is assumed that the fields of the record are in UTC.
+Along with the specified timezone, the SMT also provides configuration options to include or exclude specific fields from timezone conversion using the `include.fields` and `exclude.fields` configuration options.
+
+The SMT supports all Debezium and Kafka Connect temporal and non-temporal types.
+
+[NOTE]
+====
+To comply with daylight savings time, you must specify a geographic timezone in the `converted.timezone` configuration option.
+If you specify a UTC offset, the transform applies a fixed offset from UTC that isn't accurate for regions that observe daylight savings time.
+Providing a fixed UTC offset is useful when converting timestamp fields to a specific timezone that does not observe daylight saving time.
+====
+
+[NOTE]
+====
+The `include.fields` and `exclude.fields` configuration options are mutually exclusive. You must specify only one of the options.
+====
+
+[[timezone-converter-usage]]
+
+[[basic-example-timezone-converter]]
+== Example: Basic configuration
+
+Configure the `TimezoneConverter` SMT in the Kafka Connect configuration for a connector to convert the time-based fields in an event record to a target timezone.
+
+For example, to convert all timestamp fields in an event record from UTC to the `Pacific/Easter` timezone, add the following lines to your connector configuration:
+
+[source]
+----
+transforms=convertTimezone
+transforms.convertTimezone.type=io.debezium.transforms.TimezoneConverter
+transforms.convertTimezone.converted.timezone=Pacific/Easter
+----
+
+=== Effect of applying the `TimezoneConverter` SMT
+The following examples show the timestamp fields in an event record before and after applying the `TimezoneConverter` transformation.
```

Comment:
```suggestion
=== Effect of applying the `TimezoneConverter` SMT

The following examples show the timestamp fields in an event record before and after applying the `TimezoneConverter` transformation.
```

I think that it's best to insert a blank line between headings and the first sentence that follows. It's inconsistent, but I've seen some instances where omitting an empty line results in the first sentence being dropped when content is converted for downstream use.

Reply:
@roldanbob This creates the table of contents to be in this structure as I see in the preview:

Table of Contents
- [Example: Basic configuration](https://github.com/debezium/debezium/pull/4819#basic-example-timezone-converter)
    - [Effect of applying the TimezoneConverter SMT](https://github.com/debezium/debezium/pull/4819#_effect_of_applying_the_timezoneconverter_smt)
- [Example: Advanced configuration](https://github.com/debezium/debezium/pull/4819#advanced-example-timezone-converter)
- [Configuration options](https://github.com/debezium/debezium/pull/4819#timezone-converter-configuration-options)

Reply:
Also inside the `Effect of applying the TimezoneConverter SMT` block the headings are being shown as 
```
Example 1. Value of created_at field in an event record before processing by the TimezoneConverter transformation
``` 
and 
```
Example 2. Event record value after processing by the TimezoneConverter transformation
``` 
which I believe are part of the same example and shouldn't be depicted as Example 1 & 2.  it.

Reply:
You seemed to be saying that adding the blank line results in the TOC entry, but I don't think that's what you meant? In any case, I don't think that adding the subordinate TOC entry is a problem. 
WRT your second comment,  I suggest that we retain the _before_ and _after_ examples as separate example blocks. They are both components in demonstrating the concept of the parent heading, i.e., Effect of applying the SMT ... 

But my labeling change needs improvement. The two examples should have parallel titles. To achieve that, the first should be _"Event record value before processing by ..."_ . I've edited my original suggestion to reflect that.